# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: network-manager\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2008-08-28 09:28+0300\n"
"PO-Revision-Date: 2006-04-08 12:19+0900\n"
"Last-Translator: Michał Środek <michal@srodek.info>\n"
"Language-Team: Polish <translation-team-pl@lists.sourceforge.net>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: comariface.py:214
#, fuzzy
msgid "You are not authorized for this operation."
msgstr "Nie masz uprawnień aby zmieniać ustawienia sieci."

#: comariface.py:218 autoswitch.py:33 autoswitch.py:35 network-manager.py:35
#: network-manager.py:110 network-applet.py:615 network-applet.py:629
msgid "Network Manager"
msgstr "Manager sieci"

#: comariface.py:224 network-applet.py:176
msgid "Waiting DBus..."
msgstr "Oczekiwanie na DBus..."

#: comariface.py:224 network-applet.py:176
msgid "Connection to the DBus unexpectedly closed, trying to reconnect..."
msgstr "Połączenie z DBus niespodziewanie zostało zerwane, łączę ponownie..."

#: comariface.py:240 network-applet.py:192
msgid ""
"Cannot connect to the DBus! If it is not running you should start it with "
"the 'service dbus start' command in a root console."
msgstr ""
"Nie można połączyć się z DBus! Jeżeli nie działa to powinieneś uruchomić "
"usługę 'service dbus start' jako root."

#: comariface.py:408
msgid "new connection"
msgstr "nowe połączenie"

#: newconn.py:26
msgid "Create a new connection"
msgstr "Utwórz nowe połączenie"

#: newconn.py:31
msgid "Select device:"
msgstr "Wybierz urządzenie:"

#: newconn.py:58
msgid "Create"
msgstr "Utwórz"

#: newconn.py:112
msgid "No suitable device found"
msgstr "Nie odnaleziono odpowiednich urządzeń"

#: newconn.py:120
msgid "Install network packages!"
msgstr "Zainstaluj pakiety sieciowe!"

#: newconn.py:121
msgid "No package with COMAR network scripts are installed yet."
msgstr ""
"Żadne pakiety ze skryptami sieciowymi managera COMAR nie są jeszcze "
"zainstalowane."

#: autoswitch.py:63
msgid "Scanning..."
msgstr "Skanuję..."

#: autoswitch.py:84
#, python-format
msgid "Profile <b>%s</b> matched."
msgstr "Profil <b>%s</b> dopasowany."

#: autoswitch.py:88
msgid "There is no matched profile"
msgstr "Nie ma pasującego profilu"

#: autoswitch.py:90
msgid "No scan result"
msgstr "Brak wyników skanowania"

#: autoswitch.py:100
#, python-format
msgid "Connecting to <b>%s</b> ..."
msgstr "Łączę z <b>%s</b> ..."

#: browser.py:52
msgid "Name:"
msgstr "Nazwa:"

#: browser.py:59 connection.py:264
msgid "Authentication"
msgstr "Uwierzytelnienie"

#: browser.py:62 connection.py:495
msgid "Address:"
msgstr "Adres:"

#: browser.py:65 connection.py:536
msgid "Automatic"
msgstr "Automatycznie"

#: browser.py:72
#, fuzzy
msgid "Mod:"
msgstr "Tryb:"

#: browser.py:101
msgid "Turn on/off connection"
msgstr "Włącz/wyłącz połączenie"

#: browser.py:114
msgid "Configure connection"
msgstr "Konfiguracja połączenia"

#: browser.py:117
msgid "Delete connection"
msgstr "Usuń połączenie"

#: browser.py:139
#, python-format
msgid ""
"Should I delete the\n"
"'%s'\n"
"connection?"
msgstr ""
"Powinienem usunąć połączenie \n"
"'%s'\n"
"?"

#: browser.py:140
msgid "Delete connection?"
msgstr "Usunąć połączenie?"

#: browser.py:228
msgid "No device!"
msgstr "Brak urządzenia!"

#: browser.py:407 network-applet.py:661
msgid "New connection"
msgstr "Nowe połączenie"

#: browser.py:412 nameconf.py:28
msgid "Name Service Settings"
msgstr "Ustawienia nazwy usługi"

#: browser.py:417 network-applet.py:401
msgid "Connect Automatically"
msgstr "Połącz Automatycznie"

#: browser.py:423
#, fuzzy
msgid "Share Connection"
msgstr "Połączenie"

#: browser.py:442
msgid "Try Auto Connect on startup"
msgstr "Próbuj automatycznie połączyć się na starcie"

#: browser.py:499
msgid "Network Connections Help"
msgstr "Pomoc - połączenia sieciowe"

#: connection.py:27
msgid "Channel"
msgstr "Kanał"

#: connection.py:28
msgid "Mode"
msgstr "Tryb"

#: connection.py:29
msgid "Protocol"
msgstr "Protokuł"

#: connection.py:41
msgid "No remotes found"
msgstr "Nic nie znaleziono"

#: connection.py:61
msgid "<hidden>"
msgstr "<ukryty>"

#: connection.py:99
msgid "Scan results:"
msgstr "Wynik skanowania:"

#: connection.py:118
msgid "Scan again"
msgstr "Przeskanuj ponownie"

#: connection.py:120
msgid "Use"
msgstr "Użyj"

#: connection.py:197
msgid "Connection name:"
msgstr "Nazwa połączenia:"

#: connection.py:204
msgid "Connection"
msgstr "Połączenie"

#: connection.py:210
msgid "Device:"
msgstr "Urządzenie:"

#: connection.py:216
msgid "Select"
msgstr "Wybierz"

#: connection.py:224
#, fuzzy
msgid "Device Mode"
msgstr "Urządzenie:"

#: connection.py:230
msgid "Mode:"
msgstr "Tryb:"

#: connection.py:248
msgid "Scan"
msgstr "Skanuj"

#: connection.py:272
msgid "Security:"
msgstr ""

#: connection.py:274
msgid "No authentication"
msgstr "Brak autoryzacji"

#: connection.py:278
#, fuzzy
msgid "Authentication:"
msgstr "Uwierzytelnienie"

#: connection.py:283
#, fuzzy
msgid "Inner Authentication:"
msgstr "Uwierzytelnienie"

#: connection.py:294
msgid "Anonymous Identity:"
msgstr ""

#: connection.py:299
msgid "User/Identity"
msgstr ""

#: connection.py:304
msgid "Password:"
msgstr "Hasło:"

#: connection.py:310
msgid "Client Certificate:"
msgstr ""

#: connection.py:315
msgid "CA Certificate:"
msgstr ""

#: connection.py:320
msgid "Private Key File:"
msgstr ""

#: connection.py:325
#, fuzzy
msgid "Private Key Password:"
msgstr "Hasło:"

#: connection.py:358 connection.py:363
msgid "Certificates (*)"
msgstr ""

#: connection.py:358
msgid "Select CA Certificate File"
msgstr ""

#: connection.py:363
msgid "Select Client Certificate File"
msgstr ""

#: connection.py:368
msgid "Private Keys (*)"
msgstr ""

#: connection.py:368
msgid "Select Private Key File"
msgstr ""

#: connection.py:476
msgid "Network settings"
msgstr "Ustawienia sieci"

#: connection.py:486
msgid "Automatic query (DHCP)"
msgstr "Automatyczne zapytanie (DHCP)"

#: connection.py:491
msgid "Manual"
msgstr "Podręcznik"

#: connection.py:502 connection.py:523 connection.py:538
msgid "Custom"
msgstr "Własne"

#: connection.py:507
msgid "Net mask:"
msgstr "Maska sieci:"

#: connection.py:518
msgid "Gateway:"
msgstr "Brama:"

#: connection.py:527 nameconf.py:45
msgid "Name servers"
msgstr "Nazwy serwerów"

#: connection.py:798
msgid "Configure network connection"
msgstr "Konfiguruj połączenie sieciowe"

#: nameconf.py:35
msgid "Computer"
msgstr "Komputer"

#: nameconf.py:41
msgid "Host name:"
msgstr "Nazwa hosta"

#: nameconf.py:56
msgid "Up"
msgstr "Do góry"

#: nameconf.py:59
msgid "Down"
msgstr "Do dołu"

#: nameconf.py:171
msgid "Add Name Server"
msgstr "Dodaj Nazwę serwera"

#: nameconf.py:172
msgid "Name server:"
msgstr "Nazwa serwera:"

#: connsharing.py:108
msgid "Internet Connection Sharing"
msgstr ""

#: connsharing.py:109
#, fuzzy
msgid "Share Internet Connection"
msgstr "Utwórz nowe połączenie"

#: connsharing.py:111
msgid "Interface that goes to internet"
msgstr ""

#: connsharing.py:112
#, fuzzy
msgid "Interface that will share connection"
msgstr "Utwórz nowe połączenie"

#: connsharing.py:128
msgid "D-Bus Error"
msgstr ""

#: connsharing.py:132
msgid "COMAR Error"
msgstr "Błąd COMAR"

#: connsharing.py:136
msgid "Changing States of Interfaces Failed"
msgstr ""

#: connsharing.py:138
msgid "DHCP Server Error"
msgstr ""

#: connsharing.py:157
msgid ""
"The interfaces that you have selected must be different to share internet "
"connection"
msgstr ""

#: connsharing.py:157
msgid "Check Selected Interfaces"
msgstr ""

#: connsharing.py:167
msgid "The sharing of your connection has been successfully done."
msgstr ""

#: connsharing.py:167
msgid "Sharing is OK"
msgstr ""

#: connsharing.py:268
msgid "You do not have an ip address for the interface that goes to internet"
msgstr ""

#: connsharing.py:268
#, fuzzy
msgid "IP address not found"
msgstr "Nic nie znaleziono"

#: network-manager.py:83
msgid "Just try to connect automatically"
msgstr "Próbuj połączyć się automatycznie"

#: network-manager.py:98
msgid "Network manager module is already started!"
msgstr "Moduł zarządzania siecią jest już uruchomiony!"

#: network-applet.py:399
msgid "Firewall..."
msgstr "Firewall..."

#: network-applet.py:400
msgid "Edit Connections..."
msgstr "Edytuj połączenia..."

#: network-applet.py:403
msgid "Show Notifications"
msgstr "Pokazuj Powiadomienia"

#: network-applet.py:405
msgid "Icon Per Device"
msgstr "Ikona Urządzenia"

#: network-applet.py:406
msgid "Single Icon"
msgstr "Pojedyńcza Ikona"

#: network-applet.py:477
msgid "Should network-applet start automatically when you login?"
msgstr "Uruchamiać aplet sieci automatycznie podczas logowania?"

#: network-applet.py:592
msgid "Connecting to <b>%1</b> ..."
msgstr "Łączę z <b>%1</b> ..."

#: network-applet.py:597
msgid "Connected to <b>%1</b>"
msgstr "Połączono z <b>%1</b>"

#: network-applet.py:603
msgid ""
"Connection failed \n"
"<b>%1</b>"
msgstr ""
"Połączenie nie powiodło się\n"
"<b>%1</b>"

#: network-applet.py:605
msgid "You are offline now"
msgstr "Jesteś offline"

#: network-applet.py:699
msgid "Network Applet"
msgstr "Aplet Sieci"

#~ msgid "User name:"
#~ msgstr "Nazwa użytkownika:"
